### PR TITLE
Unify docstrings and comments

### PR DESCRIPTION
**Pull request by eugulixes**
Also
* Fix typos
* Add missing information

### CODE REVIEW

File: gits/test/terminal_test.py [hunk 1]
```diff
@@ -72,7 +73,11 @@ def _check_screen_char(self, c, pos):
         self.assertEqual(want, got)
 
     def _check_cursor_right(self, cur_x, eol=False):
-        """A helper method that checks the `_cursor_right` method."""
+        """A helper that checks the `_cursor_right` method.
+
+        The ``cur_x`` argument is the x position of the cursor.
```
Comment (dshil) [inline gits/test/terminal_test.py]: s/is the x position/is the horizontal position/
?

File: gits/test/terminal_test.py [hunk 2]
```diff
@@ -85,7 +90,12 @@ def _check_cursor_right(self, cur_x, eol=False):
             self.assertEqual(cur_x + 1, self._terminal._cur_x)
 
     def _check_cursor_down(self, cur_y, top=False):
-        """A helper method that checks the `_cursor_down` method."""
+        """A helper that checks the `_cursor_down` method.
+
+        The ``cur_y`` argument is the y position of the cursor.
```
Comment (dshil) [inline gits/test/terminal_test.py]: s/is the y position/is the vertical position/
?

**Comment by eugulixes** [comment]
@dshil yes, but unifying the signatures of the methods will be done in a different PR.

**Review by dshil — APPROVED** [review]
LGTM

**Pull request merged**